### DISCUSSION TITLE
Azimuth - AeroDyn and ElastoDyn outputs

### DISCUSSION THREAD

**Discussion by Cardoza2** [OpenFAST/openfast#1587] [Q&A] [answered]
All, 
 I've been analyzing a simplified version of the NREL 5MW (files attached below). I noticed that AeroDyn puts out interesting results for the azimuth (`B1Azimuth`) as compared to the azimuthal position from ElastoDyn (`Azimuth`). 

![azimuthplot_AD_ED](https://github.com/OpenFAST/openfast/assets/39682562/1a297219-c020-4fdd-9914-3da70a52dc68)

I recognize these are slightly different quantities, but I expected AD to return something closer to what comes out of ED. Is this what is expected? If so, what is the meaning of `B1Azimuth`? If this isn't expected, is there a setting that might make this happen? If you have any thoughts, that would be super helpful. Thanks in advance,

Adam

[sn5.zip](https://github.com/OpenFAST/openfast/files/11548038/sn5.zip)

**Comment by jjonkman** [comment, marked as answer]
Dear @Cardoza2,

The blade azimuth angle output from AeroDyn v15 is not the rigid-body rotation of the rotor, but defines the orientation of blade relative to skewed inflow–see the following topic on our forum for more information: https://forums.nrel.gov/t/azimuth-angle-of-the-blade/1464.

The Azimuth output from ElastoDyn is likely more in line with what you expect.

Best regards,

**Reply by Cardoza2** [reply]
Thanks Jason, I forgot to check the forum.
